daml/Marketplace/Custody.daml

```diff
@@ -78,7 +79,7 @@ template Service
       nonconsuming RequestLifecycle: ContractId LifecycleRequest
         with
           assetDepositCid: ContractId AssetDeposit
-          postAssetVersion: Int
+          choice: Claims
```

Comment:
How do lifecycle event work that don't require an explicit election? Like for a coupon payment, do I have to pass in the "remaining" claims for this, ie. the rhs of the `and` node?

Reply:
Yes, I didn't get around to that yet. I suspect this should be an `Optional`.